src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/set_global_properties_request.cc

```diff
@@ -317,10 +323,18 @@ void SetGlobalPropertiesRequest::on_event(const event_engine::Event& event) {
   std::string response_info;
   const bool result = PrepareResponseParameters(result_code, response_info);
 
-  SendResponse(result,
-               result_code,
-               response_info.empty() ? NULL : response_info.c_str(),
-               &(message[strings::msg_params]));
+  if (result && !is_menu_layout_available_) {
+    SendResponse(result,
+                 mobile_apis::Result::WARNINGS,
```

Comment:
You should also check to make sure that there isnt other response_info being sent back in the response. It should probably be appended to the menu layout warning or vice versa

Reply:
change made in https://github.com/smartdevicelink/sdl_core/pull/2968/commits/bac95f629eab5c798aa9031c9f0e99bb6b9aba8e